packages/styles/scss/components/data-table/_data-table-inline-edit.scss

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../utilities/button-reset';
+@use '../../utilities/convert' as *;
+@use '../../utilities/focus-outline' as *;
+
+.#{$prefix}--inline-edit-label {
```

Comment:
We might be able to ditch this style sheet for now, I'm not sure where this gets used 🤔  Do you happen to know @tw15egan ?

Reply:
I'm not too sure, I think it got ported over to Cloud PAL. It should be able to be safely removed

Reply:
Awesome! I removed it.